tests/tests_pytorch/accelerators/test_tpu.py

```diff
@@ -323,3 +323,39 @@ def test_trainer_config_device_ids(devices, expected_device_ids):
     trainer = Trainer(accelerator="tpu", devices=devices)
     assert trainer.device_ids == expected_device_ids
     assert trainer.num_devices == len(expected_device_ids)
+
+
+@RunIf(tpu=True, standalone=True)
```

Comment:
```suggestion
@RunIf(tpu=True)
```
with devices=8, the standalone mark is not necessary.